paconvert

```diff
@@ -1,4 +1,28 @@
 #!/bin/bash
+
+# _____                                    _   
+# |  __ \                                  | |  
+# | |__) |_ _  ___ ___  _ ____   _____ _ __| |_ 
+# |  ___/ _` |/ __/ _ \| '_ \ \ / / _ \ '__| __|
+# | |  | (_| | (_| (_) | | | \ V /  __/ |  | |_ 
+# |_|   \__,_|\___\___/|_| |_|\_/ \___|_|   \__|
+#
+# Copyright (C) 2020-2021
+#
+# This file is part of Paconvert
+#
+# Pacovert is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, version 3 of the License
+#
+# Pacovert is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Paconvert. If not, see <https://www.gnu.org/licenses/>.
```

Comment:
One of the reason for which I specifically said NOT to copy the boilerplate from Pacstall is that this boilerplate is specifically edited by me to be for `GPL-3.0-only`. Which for some reason you didn't follow? Anyways replace the boilerplate with the correct one:

```
    This file is part of Foobar.

    Foobar is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    Foobar is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with Foobar.  If not, see <https://www.gnu.org/licenses/>.
```

Again, it's expected that you actually read this message properly, and edit the necessary placeholders ("Foobar" etc) to the correct values.